ws/websocket.go

```diff
@@ -371,6 +396,7 @@ func (server *Server) Write(webSocketId string, data []byte) error {
 	if !ok {
 		return fmt.Errorf("couldn't write to websocket. No socket with id %v is open", webSocketId)
 	}
+	log.SendMessage(webSocketId, data)
```

Comment:
Keep in mind, that this is logged before any network operation actually occurred. It may potentially be followed by an error/reconnect issue.

Reply:
yeah, I wanted to log basically which OCPP-J messages the server/client "think" they're sending/receiving so this is fine _for my use case_, since at this point we have the `[]byte`s